src/audio.rs

```diff
@@ -257,9 +273,12 @@ impl TweenCommandKind {
     }
 }
 
+/// Cross-fade that happens at the start of the audio.
```

Comment:
Those are just marker types to reduce type duplication.
Maybe `Marker type for tween commands with fade-in/fade-out`?